docs/general/user-defined-functions.rst

```diff
@@ -105,9 +105,9 @@ You can explicitly assign a schema like this::
 Supported types
 ===============
 
-The argument types, and the return type of the function can be any of the
-CrateDB supported :ref:`data-types`. Data types of values passed into a
-function must strictly correspond to its argument data types.
+Arguments and return data types of the function can be any of the CrateDB
+supported :ref:`data-types`. Data types of values passed into the function
+must strictly correspond to its argument data types.
```

Comment:
the first sentence of this para can be simplified a little. and the second sentence can be made a little more precise. for example:

"Function arguments and return values can be any supported :ref:`data-types`. The values passed into a function must strictly correspond to the specified argument data types."